platform/docs/src/css/custom.css

```
@@ -723,6 +583,53 @@ a.dropdown__link[href='/3.9/migration-guide/3p8-to-3p9/']::after {
+    @layer docusaurus.infima {
+      all: revert-layer;
+    }
+  }
+}
+
+/* Extend isolation to Radix portal content (Dialog, Popover, Select, etc.).
+   Portals render as direct body children outside #__docusaurus and
+   .showcase-isolated, so they miss both the revert-layer and scoped preflight.
+   :where() keeps specificity at zero so Tailwind utility classes always win. */
+:where(body > :not(#__docusaurus):not(script):not(style):not(link)) {
+  font-family: 'Inter', sans-serif;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)